src/dashboard/src/requirements/production.txt

```diff
@@ -32,14 +32,16 @@ greenlet==0.4.15
 gunicorn==19.9.0
 idna==2.8
 ipaddress==1.0.22
+josepy==1.2.0
 lazy-paged-sequence==0.3
 logutils==0.3.3
 lxml==3.5.0
 metsrw==0.3.8
+mozilla-django-oidc==1.0.0
 mysqlclient==1.3.7
 ndg-httpsclient==0.5.1
-pyasn1-modules==0.2.5     # via python-ldap
```

Comment:
Where's python-ldap gone?

Reply:
It's still in the requirements. For some reason, pip-tools is reporting this dependency in a different place - I'm not sure why, possibly things being run in a different order, or a different version of pip-tools being used. The dependencies themselves look to be in order anyway :)